src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -93,7 +93,6 @@ internal MethodInfo ReplaceWithExtensionImplementation(out bool wasError)
 
                 wasError = (Method is not null && method is null) || (SetMethod is not null && setMethod is null);
 
-                // Tracked by https://github.com/dotnet/roslyn/issues/76130 : Test with indexers (ie. "method") and in compound assignment (ie. "setMethod")
```

Comment:
:memo: I added a note to indexers follow-up issue. @AlekseyTs, I assume you're already covering this in your current ref analysis work for operators